cmd/controller-manager/app/options/options.go

```diff
@@ -78,4 +80,6 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 		"The secure port on which to serve HTTPS.")
 	flags.DurationVar(&o.ClusterStatusUpdateFrequency.Duration, "cluster-status-update-frequency", 10*time.Second,
 		"Specifies how often karmada-controller-manager posts cluster status to karmada-apiserver.")
+	flags.BoolVar(&o.EnableLeaderElection, "enable-leader-election", true, "Whether you need to enable leader election.")
```

Comment:
I'm sorry, I changed my mind, I'd prefer to give the flag a new name `--leader-elect`, keeping in line with [kube-controller-manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/), so that users would feel more natural if they familiar with `kube-controller-manager`.

And the flag comments can be `Start a leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.`

Reply:
Agreed.Also keep in line with controller-runtime